.github/workflows/end-to-end-tests.yml

```diff
@@ -158,6 +158,14 @@ jobs:
       requires-aws-credentials: true
     secrets: inherit
 
+  launch_lcbench_simulated:
```

Comment:
I wonder if we could just run all examples by default, and just specific the ones we want to run on sagemaker.
Also, one thing that seems clear in addition to adding examples potentially manually is that we may want to trim them, right now there are 35 examples which is probably too much.

Reply:
This is a good suggestion. Some refactoring/cleanup will be required in order to automatically identify which examples need sagemaker versus which do not, so planning to do this in a separate PR.